.gitignore

```
@@ -5,3 +5,7 @@ debug/
 **/.DS_Store
 test/fixtures/elm/queries/translation-output
 test/fixtures/elm/queries/.start-translator
+_tests_/fhir-patient-generator/
+src/_tests_/connectathon
+src/_tests_/connectathon
+_tests_/connectathon/
```

I think this only should be root directory ones since that's where they're cloned now

```suggestion
fhir-patient-generator
connectathon
```